webclient/discovery/src/test/resources/application.yaml

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2025 Oracle and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+TestWebClientConfig-testPropertiesFromConfig:
+  base-uri: http://prod.example.com/foo
+  properties:
```

Comment:
The reason we have webclient service is to be able to configure them with the webclient in a manner that makes it clear to what the configuration belongs. This approach you use is not doing that.

Please use the same approach as with other services (both in webserver and webclient) 
I would expect the configuration to be similar to:

```yaml
webclient-config-key:
  base-uri: http://prod.example.com/foo
  services:
    discovery:
      # some structure here specific to discovery, i.e. 
      "TEST": 
          prefix-uri: "http://test.example.com"
          name: "TEST"
```

This configuration should then be processed by the service, and used when processing a request.

Reply:
So don't use client properties, but ensure this information is part of the configuration of the service itself? Just want to be clear.

Reply:
OK. I will redo the PR and documentation and specification to include this change, and will follow the suggested YAML above (or something similar to it).

Reply:
One last question: my original approach permits one-off discovery usage triggered by client properties. I'm not saying it's right, just that it's possible, which is why I went this route. For example:
```
var c = webclientbuilder.build(); // with the discovery service on/enabled/present
// For this particular request, set up discovery a particular way:
c.property("helidon-discovery-TEST-prefix-uri", "http://test.example.com/test")
  .get(someUriSomeonePassedInThatMayOrMayNotMatch);
```
(Typed without proofing but you get the idea.)

To be very clear: we want to disable this use case, correct?